desci-server/src/redisClient.ts

```diff
@@ -0,0 +1,35 @@
+import { createClient } from 'redis';
+
+const redisClient = createClient({
+  url: process.env.REDIS_URL,
```

Comment:
please make it so that if REDIS_URL is not set, or the redisClient is not working, everything still works fine